### PR TITLE
unify the two makefiles

### PR DESCRIPTION
**Pull request by acf**
The only difference between the two makefiles was the rm / delete command for clean so I just removed it for now and at least there's just a single makefile I can use on Mac and Amiga.  Attention pulled to the shiny thing of trying to make a little git client for Amiga, but hopefully will get back on track :-)

### CODE REVIEW

File: Makefile.vbcc [hunk 1]
```diff
@@ -7,15 +7,16 @@ VASM=vasmm68k_mot
...
 
-SOURCE_FILES = ObjectMemory.o ObjectMemory_Storage.o ObjectMemory_FreeList.o ObjectMemory_Allocation.o ObjectMemory_RefCounting.o ObjectMemory_GC.o RealWordMemory.o Interpreter.o Interpreter_Error.o
-HEADER_FILES = ObjectMemory.h ObjectMemory_Constants.h ObjectMemory_Storage.h ObjectMemory_FreeList.h ObjectMemory_Allocation.h ObjectMemory_RefCounting.h ObjectMemory_GC.h RealWordMemory.h Types.h Interpreter.h
-HEADER_FILES2 = Interpreter_Constants.h Interpreter_Error.h
-#had to split this because the line was too long
+OM_SOURCEFILES =  ObjectMemory.o ObjectMemory_Storage.o ObjectMemory_FreeList.o ObjectMemory_Allocation.o ObjectMemory_RefCounting.o ObjectMemory_GC.o RealWordMemory.o
+INTERP_SOURCEFILES =  Interpreter.o Interpreter_Error.o
+
+OM_HEADERFILES = ObjectMemory.h ObjectMemory_Constants.h ObjectMemory_Storage.h ObjectMemory_FreeList.h ObjectMemory_Allocation.h ObjectMemory_RefCounting.h ObjectMemory_GC.h RealWordMemory.h 
+INTERP_HEADERFILES = Types.h Interpreter.h Interpreter_Constants.h Interpreter_Error.h
```
Comment (iamleeg) [inline Makefile.vbcc]: This might be a PITA for line length but `Types.h` is used in the `ObjectMemory`.

Comment (acf) [reply Makefile.vbcc]: I noticed that, but its not like the build separate targets.  I can pull types.h into a SHARED_HEADERFILES section :-)

Comment (acf) [reply Makefile.vbcc]: I figured  better not to have a file duplicated

Comment (iamleeg) [reply Makefile.vbcc]: fair enough :)

**Pull request merged**